packages/terra-slide-panel/src/SlidePanel.jsx

```diff
@@ -5,6 +5,11 @@ import styles from './SlidePanel.module.scss';
 
 const cx = classNames.bind(styles);
 
+const position = {
```

Comment:
Leave this as strings. I don’t want to re introduced exporting enums.

Reply:
@mjhenkes 

I recommended creating an enum and exporting it assuming that it is a pattern [1] that is followed. May I know why you'd not want to re-introduce exporting enums?

[1] https://github.com/cerner/terra-core/pull/1923/

Reply:
We want to keep the propTypes as strings. You can export an enum that returns the expected strings, but we want the propTypes to stay accepting strings. To do this, we should keep the propTypes for panelPosition as `panelPosition: PropTypes.oneOf(['start', 'end']),`

Here is where we last talked about this: https://github.com/cerner/terra-core/issues/1044#issuecomment-386711960

Reply:
Let's update the `const` name to `const SlidePanelPositions` This aligns with what we talked about here: https://github.com/cerner/terra-core/issues/1044#issuecomment-386711960

Reply:
Updated "const position" to "const SlidePanelPositions" and made the necessary changes to reflect the same, i.e. PropTypes now only use strings.